thesis-de/content-de.tex

```diff
@@ -1799,94 +1816,102 @@ \subsection{Definition eines Graphen}
-  Datenvisualisierung kann entweder über einen lokalen Administrator, als auch
-  über eine LDAP\hyp{}Anbindung (Lightweight Directory Access Protocol)
-  verfügen, um hier z.B.  Unternehmensverzeichnisse anbinden zu können.
-  \1 Bei der Evaluierung der Weboberfläche muss auf die Performance der
-  Weboberfläche geachtet werden. Ein wichtiges Kernkriterium ist dabei, dass
-  der Administrator eine gewisse Zeitspanne vordefinieren kann, in welcher die
-  Graphen neu visualisiert und Daten aus dem Datenhaltungssystem abgerufen
-  werden können. Dies dient dazu, dass das Datenhaltsungssystem nicht mit
-  Anfragen überlastet wird. Ein kurzes Rechenbeispiel soll dies verdeutlichen.
-  Acht unterschiedliche User greifen gleichzeitig auf die Weboberfläche zu und
-  visualisieren hier im Fallbeispiel acht Graphen mit unterschiedlichen
-  Werten. Nun sind hier insgesamt 64 Graphen, welche sekündlich eine Abfrage an
-  das Datenhaltungssystem senden werden. Da auf das Datenhaltungssystem nicht
-  nur die Weboberfläche zugreift, sondern auch die Datenerfassung, um Daten
-  abzuspeichern, kann es hier zu massiven Performance\hyp{}Problemen kommen,
-  die durch die Weboberfläche verursacht werden können.
-  \1 Um die Sicherheit der Datenvisualisierung zusätzlich schützen zu können,
-  muss diese mit einem \GLS{API}\hyp{}System interagieren können, damit
-  eigenmächtige Modifizierungen der Daten wie das verändern, anpassen oder
-  sogar löschen unterbunden sind. Zusätzlich wird die Performance der
-  Datenvisualisierung, durch ein \GLS{API}\hyp{}System gesteigert.
-  \1 Plugin\hyp{}Erweiterung. Die Datenvisualisierung muss eine
-  Plugin\hyp{}Erweiterung unterstützen. Es dient dazu, dass im späteren
-  Verlauf, wenn das Produkt bereits beim Kunden eingesetzt wird, auf bestimmte
-  Kundenwünsche direkt eingegangen werden kann. Diese können dann mit schon zur
-  Verfügung gestellten Plugins zum Beispiel in der Community oder aber durch
-  selbst programmierte Plugins realisiert werden. Die Weboberfläche sollte
-  jedoch auch andere Sprachen unterstützen, um so eine Unabhängigkeit von
-  Entwicklern gewährleisten zu können. Unternehmen müssen so nicht auf
-  spezielle Entwickler von bestimmten Sprachen zurückgreifen.
-  \1 Die Datensicherung ist bei der Datenvisualisierung ein wesentlicher
-  Aspekt. Dabei muss die Weboberfläche bereits Möglichkeiten bieten, eine
-  Datensicherung des \glslink{Dashboard}{Dashboards} durchführen zu können. Das
-  Dashboard beinhaltet alle Graphen und vorkonfigurierte Werte. Diese sollen
-  über ein externes System (Dateifreigabeserver oder ähnlichem) abgelegt werden
-  und können somit einfach dupliziert und gesichert werden.
+  \1 Um den Zugriff und damit auch die Sicherheit des Webinterfaces zu
+  schützen, muss eine Authentisierung, Authentifizierung und Autorisierung
+  seitens von der Datenvisualisierung unterstützt werden. Diese dienen in
+  erster Linie den Daten, bei der Anmeldung am Webinterface selbst vor
```

Comment:
Diese dienen in erster Linie für den Schutz der Daten. Speziell die Anmeldung am Webinterface schützt den unautorisierten Zugriff durch Dritte.